Gemfile

```diff
@@ -59,3 +59,5 @@ gem 'net-pop'
 gem 'net-smtp'
 
 gem 'carrierwave'
+
+gem 'i18n_generators'
```

Comment:
gemのREADMEに目を通しましたか?

```ruby
Restricting the group to :development would be a good idea, since the generator would probably be used only in development mode.
```

と言及がある通り対処しましょうー

Reply:
Installationの部分をちゃんと読んでませんでした。全体に目を通すように気を付けます。
group :development内に`gem 'i18n_generators`の記述を移動させました！

Reply:
見落としてただけみたいですねー。全く読んでないのは困りますが、見た上で全部は読んでなかった、というのは仕方ないことだと思いますので、気にしすぎないで大丈夫ですよ(僕も後からよく見たら書いてあった...みたいなことよくあります :sweat_smile: )

Reply:
ありがとうございます！🙇‍♂️